spec/timezone.html

```diff
@@ -277,11 +281,11 @@ <h1>Properties of Temporal.TimeZone Instances</h1>
           </tr>
           <tr>
             <td>
-              [[OffsetNanoseconds]]
+              [[OffsetMinutes]]
             </td>
             <td>
-              An integer for nanoseconds representing the constant offset of this time zone relative to UTC, or ~empty~ if the instance represents a named time zone.
-              If not ~empty~, this value must be greater than &minus;8.64 &times; 10<sup>13</sup> nanoseconds (&minus;24 hours) and smaller than &plus;8.64 &times; 10<sup>13</sup> nanoseconds (&plus;24 hours).
+              An integer for minutes representing the constant offset of this time zone relative to UTC, or ~empty~ if the instance represents a named time zone.
+              If not ~empty~, this value must be greater than &minus;1440 minutes (&minus;24 hours) and smaller than &plus;1440 minutes (&plus;24 hours).
```

Comment:
```suggestion
              If not ~empty~, this value must be in the interval from -1440 (exclusive) to +1440 (exclusive) (i.e., strictly less than 24 hours in magnitude).
```
(refer to [interval](https://tc39.es/ecma262/multipage/notational-conventions.html#interval) for examples of "interval" variations and [Expanded Years](https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-expanded-years) for examples of signed integers)

Reply:
Cool. To make ecmarkup happy, I had to remove the `+` from the positive number. Final text: 

```
If not ~empty~, this value must be in the interval from -1440 (exclusive) to 1440 (exclusive) (i.e., strictly less than 24 hours in magnitude).
```